packages/common/src/pages/_internal/PageUiSchemaCreate.ts

```diff
@@ -10,8 +10,9 @@ import { IHubPage } from "../../core/types";
  */
 export const buildUiSchema = async (
   i18nScope: string,
-  options: Partial<IHubPage>,
-  context: IArcGISContext
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  options: Partial<IHubPage>
```

Comment:
I'd keep both `options` and `context` in here as args, but I think if you preface them with an "_", typescript won't complain about them being unused